docs/installation.md

```diff
@@ -26,26 +26,32 @@ You will need to create API Keys (Usually you get `key` and `secret`) from the E
 
 ## Quick start
 
-Freqtrade provides a Linux/MacOS script to install all dependencies and help you to configure the bot.
+Freqtrade provides the Linux/MacOS Easy Installation script to install all dependencies and help you configure the bot.
 
 !!! Note
-    Python3.6 or higher and the corresponding pip are assumed to be available. The install-script will warn and stop if that's not the case.
+    Windows installation is explained [here](#windows).
+
+The easiest way to install and run Freqtrade is to clone the bot GitHub repository and then run the Easy Installation script, if it's available for your platform.
+
+!!! Note "Version considerations"
+    When cloning the repository the default working branch has the name `develop`. This branch contains all last features (can be considered as relatively stable, thanks to automated tests). The `master` branch contains the code of the last release (done usually once per month on an approximately one week old snapshot of the `develop` branch to prevent packaging bugs, so potentially it's more stable).
+
+!!! Note
+    Python3.6 or higher and the corresponding `pip` are assumed to be available. The install-script will warn you and stop if that's not the case. `git` is also needed to clone the Freqtrade repository.
```

Comment:
The script also assumes venv ( `python -m venv`) to be available  ... it's `python3-venv` in ubuntu ... but i wouldn't point to that since it's different per OS (and python version).

Reply:
should we add this to the prerequisites too?

Reply:
well it's a prerequisite of the setup.sh script ... but not for the bot in general (i can use it in system python if i want to..)

Reply:
Got that 👍

Reply:
will add notes about python3-venv later, not in this PR
merging